packages/makeswift/src/utils/__tests__/makeswift-plugin.test.ts

```diff
@@ -0,0 +1,104 @@
+import { manipulateNextConfig } from '../manipulate-next-config'
+
+test('exporting config object', () => {
+  // Arrange
+  const input = `/**
+  * @type {import('next').NextConfig}
+  */
+ const nextConfig = {
+   /* config options here */
+ }
+
+ module.exports = nextConfig
+ `
+
+  /*
+   * There are a couple "problems" with this:
+   * 1. It adds semi-colons
+   * 2. It removes whitespace
+   *
+   * I'm going to ignore those things because I don't see clear ways to prevent that —
+   * none of that information (whether or not semi-colons existed, newlines at the end, etc.)
+   * goes into the AST, so Babel has to follow some standard. I'm going to go with the defaults.
+   */
```

Comment:
I'd be fine with that

What about the case where they haven't configured Prettier?

Reply:
I was thinking Prettier would be a dependency of our CLI and if we're running it using their API we're passing our own config via the API call. Although I'm not sure that will be a great solution either... Ideally we could preserve most of the format they have. Perhaps we can use their Prettier settings if they have them and otherwise we apply our own?

We could also not worry too much about this right now, see what it feels like when integrating and then find ways to make the CLI output and copy good so that users know they have to open the file and maybe do some formatting

Reply:
Yeah, I follow on the first part — I guess my point was in that case that we'd be arbitrarily configuring the file one way or another. It would help that subset of cases where they have it configured, though.

I say let's worry about that in another MR. I think this section of the code will have a lot of iterations